src/app/container/dockerfile/dockerfile.component.ts

```diff
@@ -48,12 +47,7 @@ export class DockerfileComponent {
   public customDownloadHREF: SafeUrl;
   public customDownloadPath: string;
   public loading = true;
-  constructor(
-    public fileService: FileService,
-    private toolQuery: ToolQuery,
-    private containerService: ContainerService,
-    private containersService: ContainersService
-  ) {
+  constructor(public fileService: FileService, private toolQuery: ToolQuery, private containersService: ContainersService) {
```

Comment:
Ditto, looks like just a formatting change

Reply:
This is actually prettier, since the length is shorter than 400 characters, it will put it on one line now